### ISSUE TITLE
Add Georgian language to staging server

### ISSUE THREAD

**Issue by arabuli** [mozilla/addons-server#2479]
Hello, we are working on localizing AMO into Georgian language. 20% is done on Pontoon, but there is no option to choose Georgian on staging server, which prevents us from testing if translated strings fit/match the context.

**Comment by andymckay** [comment]
Whats the locale code for Georgia @arabuli ?

**Comment by EnTeQuAk** [comment]
I suppose it's `ka` according to a few commits, e.g 962247863e4000a39130ba43846ccfafe013932b 121cd4df8db404c09aff765412fc4c77964dcc06 or 6b7fb8d9441a34beb2266632fe9e88b37d1858dc

I'll try and see what it takes to enable it.

**Comment by arabuli** [comment]
Its ka_GE for "Georgian (Georgia)", and ka: "Georgian".

**Comment by EnTeQuAk** [comment]
I opened up a pull request, once it lands you'll see everything on our dev server.

Unfortunately I had to open a followup ticket for `ka-GE` (https://github.com/mozilla/addons-server/issues/2739) since one package we depend on for the language name / description doesn't have `ka-GE`, once I figure out how and where to add it we'll add `ka-GE` to the list of languages as well.

**Comment by EnTeQuAk** [comment]
It should be visible on dev (https://addons-dev.allizom.org/en-US/firefox/) in the language switcher shortly now that the pull request is merged. Please feel free to reopen or report any further issues.

**Comment by ValentinaPC** [comment]
Verified as fixed on AMO-dev FF46(Win 7)
Postfix screenshot: 
![ka-ge](https://cloud.githubusercontent.com/assets/15685960/15566058/7ad2f698-2327-11e6-9b76-696dbe6cbdc8.png)
